src/System.Net.Requests/tests/HttpWebRequestTest.cs

```diff
@@ -27,7 +27,7 @@ public class HttpWebRequestTest
         private int _responseCallbackCallCount = 0;
         private readonly ITestOutputHelper _output;
 
-        public readonly static object[][] EchoServers = Configuration.Http.EchoServers;
+        public readonly static object[][] EchoServers = System.Net.Test.Common.Configuration.Http.EchoServers;
```

Comment:
You mean using ```using``` to give this an alias?  I could; didn't seem worthwhile.  FWIW, the reason for this change was that it otherwise tries to find Http in a different Configuration than the one intended and fails to do so.